src/wms/src/test/java/org/geoserver/wms/WMSTest.java

```diff
@@ -212,6 +214,36 @@ public void testCacheConfiguration() {
         assertTrue(wms.isRemoteStylesCacheEnabled());
     }
 
+    @Test(expected = ServiceException.class)
+    public void testCheckMaxDimensionsTime() throws Exception {
+        WMSInfo info = wms.getServiceInfo();
+        info.setMaxRequestedDimensionValues(1);
+        getGeoServer().save(info);
+        setupStartEndTimeDimension(
+                TIME_WITH_START_END.getLocalPart(), "time", "startTime", "endTime");
+        String name = TIME_WITH_START_END.getLocalPart();
+        MapLayerInfo mapLayerInfo = new MapLayerInfo(getCatalog().getLayerByName(name));
+        SimpleDateFormat formatter = new SimpleDateFormat("yyyy-MM-dd", Locale.ENGLISH);
```

Comment:
Good thinking about the Locale, wondering if we need to fix a timezone too (other tests seem to be doing that locally in the date format,).